src/runner/browser-env/vite/browser-modules/mocha/parser.ts

```diff
@@ -1,5 +1,6 @@
 import { ValueOf } from "type-fest";
 import { MochaEvents } from "./events.js";
+import Mocha from "mocha";
```

Comment:
I guess you're importing it just for the types to load? Can't we do `import type {} from "mocha"` then?